packages/ckeditor5-engine/src/view/renderer.ts

```diff
@@ -774,22 +766,21 @@ export default class Renderer extends ObservableMixin() {
 	 * @param actions Actions array which is a result of the {@link module:utils/diff~diff} function.
 	 * @param actualDom Actual DOM children
 	 * @param expectedDom Expected DOM children.
-	 * @param options Options
-	 * @param options.replaceText Mark text nodes replacement.
-	 * @returns Actions array modified with the `replace` actions.
+	 * @param comparator A comparator function that should return `true` if the given node is an update of the old one.
```

Comment:
I don't like this doc, its not precise and you need to know or think hard what the author meant. My proposition, something along these lines: "A comparator function that returns \`true\` if compared elements are same or similar and one can be updated to be same as the other."